src/Audit/Audit.php

```diff
@@ -167,6 +168,11 @@ public function setup(): void
      * @param  string  $location
      * @param  array<string,mixed>  $data
      * @return bool
+     *
+     * @throws AuthorizationException
+     * @throws StructureException
+     * @throws \Exception
+     * @throws \Throwable
      */
     public function log(string $userInternalId, string $userId, string $event, string $resource, string $userAgent, string $ip, string $location, array $data = []): bool
```

Comment:
Why do we have both userInternalId and userId here? this should be an Appwrite concept and utopia should be segmented from it. If we want the link to be by internalId, we should just pass that value to userId param, we don't need to maintain 2 variables.

Reply:
We will address it seperately 👌